src/fparser/two/tests/fortran2008/test_submodule_r1116.py

```diff
@@ -93,11 +99,14 @@ def test_submodule_msp(f2008_create):
       end
       ''')
     ast = Submodule(reader)
-    assert "SUBMODULE (foobar) bar\n" \
-        "  CONTAINS\n" \
-        "  SUBROUTINE info\n" \
-        "  END SUBROUTINE info\n" \
-        "END SUBMODULE bar" in str(ast)
+    expected = """
+        SUBMODULE (foobar) bar
+          CONTAINS
+          SUBROUTINE info
+          END SUBROUTINE info
+        END SUBMODULE bar
+    """
+    assert prep_expected(expected) == str(ast)
```

Comment:
NOTE: Nothing changed in the tests in this file - but while I was developing the change I did have some test failures that were extremely hard to distinguish. Changing the test to this form gives pytest a chance to give a pleasant diff (as it is comparing strings not checking string containment)